### DISCUSSION TITLE
Does compatibility mode still work for Phone (2)?

### DISCUSSION THREAD

**Discussion by juanDevelops** [SebiAi/custom-nothing-glyph-tools#25] [Q&A] [answered]
I just followed the tutorial and created a custom sound, used **compatibility mode**, everything fine but when I try to import it to composer it says that it isn't compatible with this device model. But then I tried one of the examples for the **Phone (2)** and it imported without problems.

I don't think **compatibility mode** still works.

**Comment by SebiAi** [comment, marked as answer]
Multiple people have already had this issue and I have not investigated why this happened.

As a workaround for now you can just move the audio into the `Ringtones/Compositions` folder and reopen *Glyph Composer*.
